execution/src/worker.rs

```diff
@@ -28,16 +29,17 @@ pub enum ExecutionEvent {
 /// Management commands sent to the `execution` component.
 pub enum ExecutionManagementCommand {}
 
-#[derive(Debug, Clone)]
-struct SCELedgerEntry {
-    balance: Amount,
-    bytecode: Vec<u8>,
-    data: HHashMap<Hash, Vec<u8>>,
+/// execution request
+enum ExecutionRequest {
```

Comment:
Can you format your comments as doc comments ?

Reply:
done